plots/laws/thermodynamics/dependence_of_radiation_power_on_temperature.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+from sympy import solve, symbols
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import (print_expression, units, convert_to)
+from symplyphysics.laws.thermodynamics import irradiation_of_black_body_from_temperature as stefan_boltzmann_law
+
+print(f"Formula is:\n{stefan_boltzmann_law.print_law()}")
+solved = solve(stefan_boltzmann_law.law, stefan_boltzmann_law.radiance,
+    dict=True)[0][stefan_boltzmann_law.radiance]
+radiance_temperature = solved.subs({
+    stefan_boltzmann_law.units.stefan_boltzmann_constant: 
+                            convert_to(units.stefan_boltzmann_constant,units.watt/units.meter**2/units.kelvin**4).evalf(5)
```

Comment:
Use sympy `convert_to`. This version does not support `PhysicalConstant` input.

Reply:
I'm having trouble with this. In addition, in one of the examples, "gravity constant value" was set this way. I would be grateful if you explain.